### PR TITLE
feat:[CDS-85699]: removing fabric8 from kubernetes resize and kubernetes setup step

### PR DESCRIPTION
**Pull request by GoelPratyush99**
## Describe your changes
Removed fabric8 code from deprecated kubernetes V1 code. [Comment link](https://harness.atlassian.net/browse/CDS-85699?focusedCommentId=620460)

## Steps

- Kubernetes Setup
- Kuberentes Resize

## Checklist
- [x] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [x] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/56249)

### CODE REVIEW

File: 930-delegate-tasks/src/main/java/software/wings/beans/command/ContainerResizeCommandUnit.java
[the remark this answers is not in the document]
Comment (abosii-harness) [reply 930-delegate-tasks/src/main/java/software/wings/beans/command/ContainerResizeCommandUnit.java:1]: I agree, let's just throw unsupported exception

[the remark this answers is not in the document]
Comment (abosii-harness) [reply 930-delegate-tasks/src/main/java/software/wings/beans/command/ContainerResizeCommandUnit.java:1]: Why we're still keeping this class unchanged?

**Review by abosii-harness — DISMISSED** [review]

**Review by abhinavsingh1196 — COMMENTED** [review]
> Reviewed 10 of 16 files at r1, 6 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/56249)* status: 16 file(s) to review. 1. unresolved discussions. 1 of 2 approvals obtained. Waiting on team(s): k8s-team. 

___
*[`930-delegate-tasks/src/main/java/software/wings/beans/command/ContainerResizeCommandUnit.java` line 0 at r1](https://reviewable.io/reviews/harness/harness-core/56249#gh-1415272383:-Nl3GVfe5pxrakbCl6Dy:byr7usy) ([raw file](https://github.com/harness/harness-core/blob/426e8ff3f6e68620447821b4625639aaf3329409/930-delegate-tasks/src/main/java/software/wings/beans/command/ContainerResizeCommandUnit.java#L0)):*
<details><summary><i>Previously, abosii-harness (Alexandru Bosii) wrote…</i></summary><blockquote>

Why we're still keeping this class unchanged?
</blockquote></details>

are we planning to remove this code also? 
___
*[`930-delegate-tasks/src/main/java/software/wings/beans/command/KubernetesSetupCommandUnit.java` line 39 at r2](https://reviewable.io/reviews/harness/harness-core/56249#-Nl3Gquc-YvZ4EDjwNnk:-Nl3Gqud0Eo8fFPrmV-7:beitbhs) ([raw file](https://github.com/harness/harness-core/blob/7ffb1509916376bbebdd3b386e4062de5454fe29/930-delegate-tasks/src/main/java/software/wings/beans/command/KubernetesSetupCommandUnit.java#L39)):*
> ```Java
> public class KubernetesSetupCommandUnit extends ContainerSetupCommandUnit {
>   public KubernetesSetupCommandUnit() {
>     super(CommandUnitType.KUBERNETES_SETUP);
> ```

nit: maybe throw unsupported exception here also?

**Review by abhinavsingh1196 — DISMISSED** [review]
> Reviewed 6 of 6 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/56249)* status: 16 file(s) to review. Waiting on team(s): k8s-team.

**Comment by GoelPratyush99** [comment]
trigger uts

**Review by abosii-harness — APPROVED** [review]

**Comment by GoelPratyush99** [comment]
trigger uts

**Comment by GoelPratyush99** [comment]
trigger ut5

**Comment by GoelPratyush99** [comment]
trigger uts

**Review by abosii-harness — APPROVED** [review]

**Comment by GoelPratyush99** [comment]
trigger review

**Comment by abosii-harness** [comment]
@acasian-harness this has been deprecated for more than year and all customers should have been moved to k8s v2

**Comment by acasian-harness** [comment]
great, thanks @GoelPratyush99

**Pull request merged**